### DISCUSSION TITLE
XML file

### DISCUSSION THREAD

**Discussion by Cheeevp** [3kh0/ext-remover#558] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there a way to play geforce on a proxy without it saying This XML file does not appear to have any style information associated with it? The document tree is shown below.
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>YHMHSC89HFHS1SG7</RequestId>
<HostId>WVDF6BPK2n/KHo6zmX6vYKxINLFfrqSoV2uCxJcxb9HzNC9+Pi7D6su1ntmS7wwzz5sz/FEH1+Y=</HostId>
</Error>

**Comment by Cheeevp** [comment]
And I know Geforce works on this proxy because I tried it on  windows chrome and it worked but it doesn't work on Chromebook chrome

**Reply by Aka-but-nice** [reply, marked as answer]
Too much requests.

**Comment by amelia-bartlett** [comment]


**Reply by Aka-but-nice** [reply]
This is true, similar to the answer I said, this is the reason, once a server reached its limit, that'll cause high-performance sites to not function, in other words, Stop working.
